.cspell.json

```diff
@@ -143,5 +143,8 @@
     "llms",
     "vcrpy"
   ],
+  "flagWords": [
+    "Prompt Flow"
```

Comment:
looks like we cannot detect case like "Prompt flow" which not starts a sentence?

Reply:
Yep, did not find a way to check that..